### ISSUE TITLE
A11y - Filter/View/Download buttons are not keyboard operable

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#312]
## Describe the task
Users should be able to navigate to the buttons via TAB and when they press ENTER, the button activates, same as clicking with a mouse.

## Links
PR: https://github.com/CodeForPhilly/vacant-lots-proj/pull/349
